molo/core/api/importers.py

```diff
@@ -488,13 +492,38 @@ def attach_social_media_image(self, page, foreign_image_id):
         except (KeyError, ObjectDoesNotExist):
             pass
```

Comment:
@nathanbegbie maybe lets add todos in places like this, so when you are doing the PR for handling the errors we won't forget about them